stable/traefik/templates/configmap.yaml

```diff
@@ -353,3 +353,13 @@ data:
         {{- end }}
       {{- end }}
     {{- end }}
+    [respondingTimeouts]
```

Comment:
These should be optional settings, for backwards compatibility. You will need to test if they exist, and use them if they do, instead of testing if they are empty.